jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/RunCommandDialog.kt

```diff
@@ -32,7 +31,7 @@ import javax.swing.JComponent
 import javax.swing.JTextField
 import javax.swing.plaf.basic.BasicComboBoxEditor
 
-class RunCommandDialog(private val project: Project, private val container: ContainerDetails) :
+class RunCommandDialog(private val project: Project, private val container: ContainerDetails, private val connectionSettings: Map<String, String>) :
```

Comment:
Callers of this should not care about us turning the creds into env vars, I would just take the ConnectionSettings here